search/v2/geocoder.cpp

```diff
@@ -609,6 +659,44 @@ void Geocoder::FillLocalitiesTable(MwmContext const & context)
   }
 }
 
+void Geocoder::FillVillageLocalities()
+{
+  vector<Locality> preLocalities;
+  FillLocalityCandidates(m_villages, kMaxNumVillages, preLocalities);
+
+  size_t numVillages = 0;
+
+  for (auto & l : preLocalities)
+  {
+    FeatureType ft;
+    m_context->m_vector.GetByIndex(l.m_featureId, ft);
+
+    switch (m_model.GetSearchType(ft))
+    {
+    case SearchModel::SEARCH_TYPE_CITY:
+    {
+      if (numVillages < kMaxNumVillages && ft.GetFeatureType() == feature::GEOM_POINT)
+      {
+        ++numVillages;
+        City village = l;
```

Comment:
It's the only place where it is used.
1. Type allias would be called TVillage.
2. A separate type would infer code duplication.